File: JavaAssignment/Product.java
package com.sonata.assi;

public class Product {
	int proid;
	String proname;
	double proprice;
	
	Product(int a,String b,double c){
		this.proid=a;
		this.proname=b;
		this.proprice=c;
	}
	
	public double GST(int a) {
		return this.proprice+a;
	}
	
	public void display() {
		System.out.println(proid);
		 System.out.println(proname);
		 
	}
	
public static void main(String args[]) {
	Product p=new Product(001, "Clutches", 2500);
	p.display();
	System.out.println();
	System.out.println("The price of product inclusive GST is "+p.GST(70));
	
}
}

File: TypeScript/Product.ts
class Product{

    pId:number=11;

    pName: string = "Refridgerator"

    pPrice: number= 400000;
    
    constructor(i:number, n:string, b:number){

        this.pId=i;

        this.pName=n;

        this.pPrice=b;

    }

    Price():number{

       let gst:number= 100;

        return this.pPrice = this.pPrice+gst;

    }

    display(){

        console.log(this.pId);

        console.log(this.pName);

        console.log(this.pPrice);

    }

}

var f1 = new Product(11,"Refridgerator",400000);

var u= f1.Price();

f1.display();